src/stats/common.ts

```diff
@@ -137,6 +137,18 @@ export enum State {
   FALL_BACKWARD = 0x1f,
   GRAB = 0xd4,
   CLIFF_CATCH = 0xfc,
+  DAMAGE_FALL = 0x26,
+
+  // Command Grabs
+  BARREL_CANNON_WAIT = 0x153,
```

Comment:
Did you mean to explicitly ignore the BarrelWait or BarrelCannonWait?

```suggestion
  BARREL_WAIT = 0x125,
```